pkg/util/tracing/collector/collector.go

```diff
@@ -50,9 +49,11 @@ func New(
 	}
 }
 
-// Iterator can be used to return RecordedSpans from all live
-// nodes in the cluster, in a streaming manner. The iterator buffers the
-// RecordedSpans of one node at a time.
+// Iterator can be used to return tracing.Recordings from all live nodes in the
+// cluster, in a streaming manner. The iterator buffers the tracing.Recordings
+// of one node at a time.
+// A tracing.Recording is defined as span recordings for all
```

Comment:
This tracing.Recording comment is better paced near the type itself.

Reply:
added a little bit to the comment near the type.